homeassistant/components/integration/sensor.py

```diff
@@ -66,22 +65,18 @@
 
 DEFAULT_ROUND = 3
 
-PLATFORM_SCHEMA = vol.All(
-    cv.deprecated(CONF_UNIT_OF_MEASUREMENT),
-    PLATFORM_SCHEMA.extend(
-        {
-            vol.Optional(CONF_NAME): cv.string,
-            vol.Optional(CONF_UNIQUE_ID): cv.string,
-            vol.Required(CONF_SOURCE_SENSOR): cv.entity_id,
-            vol.Optional(CONF_ROUND_DIGITS, default=DEFAULT_ROUND): vol.Coerce(int),
-            vol.Optional(CONF_UNIT_PREFIX, default=None): vol.In(UNIT_PREFIXES),
-            vol.Optional(CONF_UNIT_TIME, default=TIME_HOURS): vol.In(UNIT_TIME),
-            vol.Optional(CONF_UNIT_OF_MEASUREMENT): cv.string,
```

Comment:
Keep it as `vol.Removed(CONF_UNIT_OF_MEASUREMENT)` (so its cleared from the result